### PR TITLE
README updates in prep for blog post on rhel entitlements

### PR DESCRIPTION
**Pull request by gabemontero**

### CODE REVIEW

**Comment by gabemontero** [comment]
@bparees @siamaksade @sbose78 @adambkaplan 

per our discussion in https://docs.google.com/document/d/1CGQpvo3JbKSdWbJJ_o29jOkld2YX4r5eXdXny4gY0d8/edit# 
this is the update to this REPO's readme to pull out the tl;dr detail from the blog draft and place it here.  The blog post can then link to this section of the README when it goes official

if you all have cycles PTAL / review / propagate this to lgtm mode

**Comment by sbose78** [comment]
Cool, thank you. Will do!

**Comment by bparees** [comment]
I didn't peruse it in detail for accuracy, but at a high level it looks good to me, i really like the way it outlines the scenarios and expected outcome for each.  good stuff.

File: deploy/csi-hostpath-plugin.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ spec:
       serviceAccountName: csi-driver-projected-resource-plugin
       containers:
         - name: node-driver-registrar
-          image: quay.io/openshift/origin-csi-node-driver-registrar:latest
+          image: quay.io/openshift/origin-csi-node-driver-registrar:4.7.0
```
Comment (sbose78) [inline deploy/csi-hostpath-plugin.yaml:38]: 👍

File: deploy/csi-hostpath-plugin.yaml [hunk 2]
```diff
@@ -60,7 +60,8 @@ spec:
               name: csi-data-dir
 
         - name: hostpath
-          image: quay.io/openshift/origin-csi-driver-projected-resource:latest
+          # precise version of the 4.8.0 tag used for the blog post branch/tag
+          image: quay.io/openshift/origin-csi-driver-projected-resource@sha256:2007468d111cdb9bd313087c8cf267d584c4bd7489f558f4a29eb3466325b85c
```
Comment (sbose78) [inline deploy/csi-hostpath-plugin.yaml:64]: 👍

File: README.md [hunk 3]
```diff
@@ -296,3 +331,124 @@ $ oc delete -f ./examples
 
 And the edit `./examples/02-csi-share.yaml` and change the `backingResource` stanza to point to the item 
 you want to share, and then re-run `oc apply -f ./examples`.
+
+## Reference to our blog on using this component to leverage RHEL Entitlements
+
+The blog post is under internal development.  A link to it will be added here when it is available.
+
+## Some more detail and examples around those vetted scenarios
```
Comment (sbose78) [inline README.md:339]: Just call it "Scenarios"? :)

**Review by sbose78 — APPROVED** [review]
LGTM!

**Comment by sbose78** [comment]
Didn't expect prow to do that 🤦

**Pull request merged**